### PR TITLE
Added multiple files options to `UseDefaultConfigs`

### PR DESCRIPTION
**Pull request by tmenezes**
- Added an overload in order to allow client apps to fully control the filenames and their respective order to be added into configuration builder.

### CODE REVIEW

**Review by tabman83 — CHANGES_REQUESTED** [review]
you need to update package v number and release notes otherwise the package won't be published by the release pipeline

File: src/Eshopworld.DevOps/Extensions/ConfigurationExtensions.cs [hunk 1]
```diff
@@ -175,6 +175,31 @@ public static IConfigurationBuilder UseDefaultConfigs(this IConfigurationBuilder
...
+        /// - 1. Environment variables
+        /// - 2. Command line arguments
+        /// - 3. Json files respection the order
+        /// Note:
+        /// - Each configuration json file WILL be added to configuration builder respecting the given sort order.
+        /// </summary>
+        /// <param name="builder">The configuration builder to bind to.</param>
+        /// <param name="appSettingsFiles">The application settings files paths.</param>
+        /// <returns>The configuration builder after config has been added.</returns>
+        public static IConfigurationBuilder UseDefaultConfigs(this IConfigurationBuilder builder, string[] appSettingsFiles)
```
Comment (goenning) [inline src/Eshopworld.DevOps/Extensions/ConfigurationExtensions.cs:190]: I think the other UseDefaultConfigs should could this method to remove some of the duplication on AddEnvironmentVariables and AddCommandLine

Comment (goenning) [reply src/Eshopworld.DevOps/Extensions/ConfigurationExtensions.cs:190]: other than that, looks good to me

File: src/Eshopworld.DevOps/Extensions/ConfigurationExtensions.cs [hunk 2]
```diff
@@ -175,6 +175,31 @@ public static IConfigurationBuilder UseDefaultConfigs(this IConfigurationBuilder
...
+        {
+            builder.AddEnvironmentVariables()
+                   .AddCommandLine(Environment.GetCommandLineArgs());
+            
+            foreach (var file in appSettingsFiles ?? throw new ArgumentNullException(nameof(appSettingsFiles)))
+            {
+                builder.AddJsonFile(file, true);
```
Comment (ppiriz) [inline src/Eshopworld.DevOps/Extensions/ConfigurationExtensions.cs:197]: will we need to compose and send all 4 json files name?
i.e. 
"appsettings.json"
"appsettings.{env}.json" 
"appsettings2.json"
"appsettings2.{env}.json"

or can we send two file names and the specific environment will be added as well? we expect to be json files as we are using AddJsonFile.
i.e.
"appsettings"
"appsettings2"
and will result in same as above in the order:
"appsettings.json"
"appsettings.{env}.json" 
"appsettings2.json"
"appsettings2.{env}.json"

Comment (rmccabe24) [reply src/Eshopworld.DevOps/Extensions/ConfigurationExtensions.cs:197]: If it was me - I'd possibly keep it simple and just send the explicit names when using the overload :-)

File: src/Eshopworld.DevOps/Extensions/ConfigurationExtensions.cs (hunk 1 above)
Comment (tmenezes) [reply src/Eshopworld.DevOps/Extensions/ConfigurationExtensions.cs:188]: Just refactored the code

**Review by tabman83 — APPROVED** [review]

**Pull request merged**